samcli/lib/init/templates/cookiecutter-aws-sam-hello-dotnet/cookiecutter.json

```diff
@@ -1,4 +1,4 @@
 {
     "project_name": "Name of the project",
-    "runtime": "dotnetcore2.1"
+    "runtime": "dotnetcore3.1"
```

Comment:
Should we keep this as `dotnetcore2.1` for backwards compatibility?

Reply:
`dotnetcore2.1` is no longer supported by Lambda. https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html